pkg/cmd/pulumi/new_ai.go

```diff
@@ -159,6 +159,40 @@ func sendPromptToPulumiAI(
 	return projectURL, connectionID, conversationID, nil
 }
 
+const initialPromptPrompt = `What cloud infrastructure would you like to build?
+(try something like "a static website on AWS behind a CDN")`
+
+const refinePromptPrompt = "Tell Pulumi AI how to refine the previous program:"
+
+var errEmptyPrompt = errors.New("prompt cannot be empty")
+
+func isValidAiPrompt(prompt string) error {
+	if prompt == "" {
+		return errors.New("prompt cannot be empty")
```

Comment:
Should this be returning `errEmptyPrompt`?

Reply:
Yeah.